integration/spark/spark35/src/main/java/io/openlineage/spark35/agent/lifecycle/plan/CreateReplaceOutputDatasetBuilder.java

```diff
@@ -63,6 +63,7 @@ public boolean isDefinedAt(SparkListenerEvent event) {
 
   @Override
   protected List<OpenLineage.OutputDataset> apply(SparkListenerEvent event, LogicalPlan plan) {
+    log.info(String.format("Applying CreateReplaceOutputDatasetBuilder to %s", plan));
```

Comment:
`{}` instead of String.format for lazy formatting, also do we need info level here?